Main.py

```diff
@@ -515,27 +515,27 @@ def on_prepare(event):
                 res = runShell(theCmd)
                 # expect 0
                 if res.returncode != 0:
-                    print("ERROR: boot.img is not Found!")
+                    print("ERROR: boot.img is not found!")
                     print(res.stderr)
                     print("Aborting ...")
                     del wait
                     return
 
                 # See if magisk tools is installed
-                print("Checking to see if Magisk Tools is installed on the phone ...")
+                print("Checking to see if Magisk tools is installed on the phone ...")
                 theCmd = self._config.adb + " -s " + self._config.adb_id + " shell \"su -c \'ls -l /data/adb/magisk/\'\""
```

Comment:
"Magisk Tools" is the name of the tool
https://topjohnwu.github.io/Magisk/tools.html

Reply:
I see